components/access-management/kfam/api_default.go

```diff
@@ -42,17 +45,17 @@ type KfamV1Alpha1Interface interface {
 type KfamV1Alpha1Client struct {
 	profileClient ProfileInterface
 	bindingClient BindingInterface
-	clusterAdmin []string
-	userIdHeader string
-	userIdPrefix string
+	clusterAdmin  []string
+	userIdHeader  string
+	userIdPrefix  string
```

Comment:
Is there a reason for double spaces here?

Reply:
I don't know, my believe that go linter did that change.

Reply:
[gofmt](https://golang.org/cmd/gofmt/) would align them, and that is usually used in go projects